bench/ibench/autovac_driver.py

```diff
@@ -174,9 +194,19 @@ def doVacuum(self):
 class SimulatedVacuum:
     def startExp(self, env_info):
         self.env_info = env_info
-        self.initial_size, self.update_speed = getParamsFromExperimentId(self.env_info['experiment_id'])
+        collectExperimentParams(self.env_info)
+        self.initial_size = self.env_info['initial_size']
```

Comment:
Same here. The simulator (this instance) does not need those.
Consider creating a difference simulator instance. It will be ideal to also have abstract class to derive from as I mentioned above.